spec/struct.dd

```diff
@@ -1106,6 +1106,26 @@ $(H3 $(LNAME2 field-init, Field initialization inside a constructor))
 
 $(H2 $(LEGACY_LNAME2 StructCopyConstructor, struct-copy-constructor, Struct Copy Constructors))
 
+    $(P $(RED Warning:) The plan is for
+    $(RELATIVE_LINK2 struct-copy-constructor, copy constructors) to replace
+    $(RELATIVE_LINK2 struct-postblit, postblit constructors), leaving postblit
+    constructors only in legacy code. However, because the compiler hooks in
+    druntime which deal with dynamic arrays and associative arrays have not yet
+    all been updated to support copy constructors properly
+    $(HTTPS github.com/dlang/dmd/issues/20970, issue #20970), any type that might
+    be used in a dynamic array or associative array should use a postblit
+    constructor rather than a copy constructor. The copy constructor will not
+    be called in all the cases where it should be for the elements of a dynamic
+    array or the keys or values in an associative array. Postblit constructors
+    do not have this problem.)
+
+    For backward compatibility reasons, a `struct` that explicitly defines both
+    a copy constructor and a postblit will only use the postblit for implicit
+    copying. However, if the postblit is disabled, the copy constructor will be
```

Comment:
Would mentioning the attribute explicitly be more comprehensible?
i.e.

> […] However, if the postblit is `@disabled`, […]

Reply:
I don't know why it would be more comprehensible. It would also arguably be wrong if you want to be really technical, because it's possible for a postblit constructor to be disabled for other reasons. The one that comes to mind at the moment would be if one or more of the member variables has had its postblit constructor disabled, and in such a case, the type wouldn't have necessarily declared any postblit or copy constructors (and it would need to declare one to be copyable).